tests/integration/chargebacks.test.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import httpAdapter from 'axios/lib/adapters/http';
+import dotenv from 'dotenv';
+
+import mollie from '../../dist/mollie';
+
+/**
+ * Overwrite the default XMLHttpRequestAdapter
+ */
+axios.defaults.adapter = httpAdapter;
+
+/**
+ * Load the API_KEY environment variable
+ */
+dotenv.config();
+
+const mollieClient = mollie({ apiKey: process.env.API_KEY });
+
```

Comment:
@vernondegoede in order to start debugging this you will have to print two things here:
* All environment variables this application can read, this way we know if `API_KEY` is set at all in the pipeline (even though the settings say it is)
* The full mollieClient object, so we can see if the apiKey variable is set if the env variable does exist

Reply:
I don't think we ever set an `API_KEY` environment variable in Travis config (using a encrypted variable): https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml

@dabroek Have you ever get this to work on the `mollie:next` branch?

Reply:
Yes I had it working before. I put the environment variable there myself.

Reply:
I could see the `API_KEY` in the settings, except for the value assigned to it.